src/lab4.py

```diff
@@ -0,0 +1,52 @@
+class WaterPump:
+
+    def __init__(self, power_consumption: int = 0, brand: str = "", volume_ph: int = 0, price: float = 0.0, name: str = ""):
+        self.__power_consumption = power_consumption 
+        self.__brand = brand
+        self.__volume_ph = volume_ph
+        self.price = price
+        self.name = name
+
+
+    
+    def get_power_consumption(self)-> int :
+        return self.__power_consumption
+    def get_brand(self)-> str :
+        return self.__brand
+    def get_volume_ph(self)-> int :
+        return self.__volume_ph
+
+
+
+    def __str__(self)-> str :
+        return f"""Power consumption = {self.__power_consumption} W,
+Brand = {self.__brand},
+Volume per hour = {self.__volume_ph} L/h,
+Price = {self.price} $,
+Name = {self.name}"""

```

Comment:
shift those elements at 4 spaces righter than the return statement for better readability

Reply:
Done!